cypress/support/commands.js

```diff
@@ -63,3 +63,11 @@ Cypress.Commands.add("getInputByLabel", (label) =>
       cy.get(`#${id}`);
     })
 );
+
+Cypress.Commands.add("openTableFilter", (columnClassName) =>
+  cy
+    .get(
```

Comment:
Selecting on Column Text to find the filter button won't work in this case since the filter button is not a descendant of column text and cy.find is only capable of searching descendants. Since we can't target the column text directly, we could try telling this selector the column #  instead of targeting a className that resembles dataCy.